cli/config/networkconfig.cpp

```diff
@@ -0,0 +1,43 @@
+#include "networkconfig.h"
```

Comment:
👍 nice for adding this.

However, `NetworkConfig` has a slightly different meaning for the Elrond Gateway, for ERDJS and ERDPY (might generate a bit of confusion). That is, `NetworkConfig` generally means "the structure returned by the API endpoint `network/config`".

Perhaps `AppConfig`?

Reply:
> +1 nice for adding this.

Thank you ! :)

> However, NetworkConfig has a slightly different meaning for the Elrond Gateway, for ERDJS and ERDPY (might generate a bit of confusion). That is, NetworkConfig generally means "the structure returned by the API endpoint network/config".

Oh, ok, that's my bad, wasn't aware of it

> Perhaps AppConfig?

I think CLIConfig would work in this case, since this config is only meant for CLI